src/main/java/mate.academy/lib/Configuration.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.lib;
+
+import java.util.HashMap;
+import java.util.Map;
+import mate.academy.service.FileReaderService;
+import mate.academy.service.ProductParser;
+import mate.academy.service.ProductService;
+import mate.academy.service.impl.FileReaderServiceImpl;
+import mate.academy.service.impl.ProductParserImpl;
+import mate.academy.service.impl.ProductServiceImpl;
+
+public class Configuration {
```

Comment:
Configuration for what? Bad naming of class.
```suggestion
public class Configuration {
```